src/helpers/actions/deployCommands.ts

```diff
@@ -16,12 +17,17 @@ export async function deployCommands(client: Client<true>): Promise<void> {
 	logger.info(`Syncing ${commands.length} command(s)...`);
 
 	const guildCommands: Array<GuildedCommand> = [];
-	const globalCommands: Array<GlobalCommand> = [];
+	const globalCommands: Array<GlobalCommand | ContextMenuCommand> = [];
 	for (const cmd of commands) {
-		if (cmd.requiresGuild) {
-			guildCommands.push(cmd);
-		} else {
+		if (
+			('type' in cmd &&
+				(cmd.type === ApplicationCommandType.Message ||
```

Comment:
Why are context menu commands(?) always global?

Reply:
Because I can't be bothered to make guild-only ones yet 😅. I literally only need this type tree for this one command, and this command can run anywhere. We might not even do more context menu commands, let alone ones that commonly need guild guarantees, certainly not when our main focus is parity with Ze Kaiser.